packages/design-system/src/css/_tokens.dark.scss

```diff
@@ -313,6 +313,8 @@
 	}
 }
 
-body.theme-dark {
-  @include theme;
+@media (prefers-color-scheme: dark) {
+	body.dark-mode-beta {
```

Comment:
I like that you added `beta`? but can you keep the same pattern at least.. `theme-dark-beta`.. to allow us to introduce new themes easily